src/qinfer/tomography/models.py

```diff
@@ -88,6 +88,9 @@ def __init__(self, basis, allow_subnormalized=False):
         self._allow_subnormalied = allow_subnormalized
         super(TomographyModel, self).__init__()
 
+        # TODO: allow more outcomes
+        self._domain = IntegerDomain(min=0, max=0)
```

Comment:
Sorry for my not fully understanding yet, but does this contradict the definition of ``n_outcomes`` below? My impression had been that ``FiniteOutcomeModel`` uses ``n_outcomes`` to set ``domain``, but this snippet and L137 below seem to be defining ``domain`` as returning copies of ``IntegerDomain(0, 0)`` (1-outcome models).